source/jobs/data-verification/data-verification.txt

```diff
@@ -2,6 +2,35 @@
 Data Verification
 =================
 
+Data verification is a process which is built into the sync engine.
+The data verification process uses the source and destination 
+connection strings to validate that a sync job was successful.
+
+- Data verification is supported for one-time sync jobs.
+- By default, data verification is not performed when running a sync job. 
+  You must explicitly opt-in to verification in :guilabel:`Migration Options` 
+  before the job is started.
+- Enabling data verification can add significant compute time to a sync 
+  job.
+
+When data verification is successful, source and destination states match.
+
+- The source relational database row count equals the target collection 
+  document count.
+- The source relation rows are equivalent to the collection documents.
+
+When data verification is unsuccessful, source and destination states 
+do not match.
+
+Data verification does not consider the following when verifying 
+destination databases:
+
+- The destination collection contained unrelated data before the sync 
+  job started.
+- The data in the relational database tables has changed since the sync 
+  job was started.
+
```

Comment:
(info typing)
Use a Learn More heading and then include just a link to the Use Data Verification page, omitting the additional text.

Reply:
Updated.